### PR TITLE
Fixes #10289: default root password hash is SHA256 now

### PR DESCRIPTION
**Pull request by dmitri-d**

### CODE REVIEW

File: app/services/password_crypt.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 require 'base64'
 
 class PasswordCrypt
-  ALGORITHMS = {'MD5' => '$1$', 'SHA256' => '$5$', 'SHA512' => '$6$', 'Base64' => ''}
+  ALGORITHMS = {'SHA256' => '$5$', 'SHA512' => '$6$', 'MD5' => '$1$', 'Base64' => ''}
```
Comment (dLobatog) [inline app/services/password_crypt.rb:4]: Curious, why changing the order?

Comment (dmitri-d) [inline app/services/password_crypt.rb:4]: More of a hint, really, that SHA256 is default/preferred.

**Comment by dLobatog** [comment]
Set to WoC as tests are red.

**Comment by dmitri-d** [comment]
[test]

**Comment by dmitri-d** [comment]
@eLobato: tests passed (was a false negative the first time).

**Comment by dLobatog** [comment]
Merged as 6c19571bd35d4c6e2a971ee707e2bc35afc8f461, thanks @witlessbird!

**Pull request closed without merging**